data/source.txt

```diff
@@ -0,0 +1,4 @@
+This is a first test of the sentiment analyzer.
```

Comment:
Is this the text you are going to analyze for sentiment in an app you are working on?  If so, can you store it in https://github.com/stanford-ppl/test-data?

Reply:
Done.